package.json

```diff
@@ -46,7 +46,7 @@
     "karma-opera-launcher": "^1.0.0",
     "karma-phantomjs2-launcher": "^0.5.0",
     "karma-safari-launcher": "^1.0.0",
-    "karma-sauce-launcher": "^1.1.0",
+    "karma-sauce-launcher": "^2.0.2",
```

Comment:
Upgrading this dependency fixed this problem which I was also able to see locally: https://travis-ci.org/stacktracejs/error-stack-parser/builds/566016807#L620